.circleci/config.yml

```diff
@@ -12,3 +45,19 @@ workflows:
           context:
             - snyk-bot-slack
           channel: group-cloud-security-vulnerabilities-alerts
+      - scan-code:
+          name: Scan Code
+          context:
+            - tiki-snyk
+          filters:
+            branches:
+              ignore:
+                - main
+      - scan-deps:
+          name: Scan Dependencies
+          context:
+            - tiki-snyk
+          filters:
+            branches:
+              ignore:
+                - main
```

Comment:
I don't understand why this bit is here (same for scan code above), but it's not a blocker.

Reply:
Copy-pasta from elsewhere to be honest. It seems like a bit of a premature optimisation, I'll remove it.